clippy_lints/src/operators/absurd_extreme_comparisons.rs

```diff
@@ -119,6 +119,19 @@ fn detect_absurd_comparison<'tcx>(
 }
 
 fn detect_extreme_expr<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'_>) -> Option<ExtremeExpr<'tcx>> {
+    // Detect Duration::from_secs(0), which is an extreme value because
+    // Duration's internal representation ensures it can never be less than zero
+    if let ExprKind::Call(_, args) = &expr.kind
+        && args.len() == 1
```

Comment:
You can use something like:

```suggestion
    if let ExprKind::Call(_, [arg]) = &expr.kind
```